### PR TITLE
Add support for customBlockFn

### PR DESCRIPTION
**Pull request by mattmcginnis**

### CODE REVIEW

File: lib/lib/replaceTextWithMeta.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ Object.defineProperty(exports, "__esModule", {
 });
 exports.default = replaceTextWithMeta;
 function replaceTextWithMeta(subject, searchText, replaceText) {
-  var text = subject.text;
-  var characterMeta = subject.characterMeta;
+  var text = subject.text,
+      characterMeta = subject.characterMeta;
```
Comment (cachilders) [inline lib/lib/replaceTextWithMeta.js:9]: Not sure how necessary this one is.

File: lib/stateFromElement.js [hunk 2]
```diff
@@ -63,9 +61,9 @@ var getEntityData = function getEntityData(tagName, element) {
   if (ELEM_ATTR_MAP.hasOwnProperty(tagName)) {
     var attrMap = ELEM_ATTR_MAP[tagName];
     for (var i = 0; i < element.attributes.length; i++) {
-      var _element$attributes$i = element.attributes[i];
-      var name = _element$attributes$i.name;
-      var value = _element$attributes$i.value;
+      var _element$attributes$i = element.attributes[i],
+          name = _element$attributes$i.name,
+          value = _element$attributes$i.value;
```
Comment (cachilders) [inline lib/stateFromElement.js:66]: Is your IDE doing this? We don't do this, generally. Or is this just the updated code from the main lib?

**Review by cachilders — APPROVED** [review]
I'm assuming the purely aesthetic changes in this PR are copied directly from later versions of the forked lib. If I'm wrong and they are automatic changes from the editor we should probably get rid of them.

File: lib/lib/replaceTextWithMeta.js (hunk 1 above)
Comment (rahuljpgupta) [reply lib/lib/replaceTextWithMeta.js:9]: files under lib/* are generated by Babel. There was no manual change made on these files.

Comment (cachilders) [reply lib/lib/replaceTextWithMeta.js:9]: lol. thanks.

File: lib/stateFromElement.js (hunk 2 above)
Comment (rahuljpgupta) [reply lib/stateFromElement.js:66]: Same as above, babel generated this file. Doing any manual change here would not be a good idea.

**Comment by jasonenglish** [comment]
still subscribed to this repo and saw the emails come in. thinking of you all! hope you're safe!

giphy-1

giphy-2

**Pull request merged**